docs/source/openapi.json

```diff
@@ -0,0 +1,234 @@
+{
+  "openapi" : "3.0.0",
+  "info" : {
+    "version" : "0.0.1",
+    "title" : "Amastra samples",
+    "description" : "",
+    "contact" : {
+      "email" : "dama@bnl.gov"
+    },
+    "license" : {
+      "name" : "Apache 2.0",
```

Comment:
The license should be BSD3?

Reply:
Go catch. How do we choose license? Do we prefer BSD3 or it's just group/org decision?